XenAdmin/Wizards/DRWizards/DRFailoverWizard.cs

```diff
@@ -83,7 +83,7 @@ public DRFailoverWizard(Pool pool, DRWizardType wizardType)
             WizardType = wizardType; 
 
             #region RBAC Warning Page Checks
-            if (Pool.Connection.Session.IsLocalSuperuser || Helpers.GetMaster(Pool.Connection).external_auth_type == Auth.AUTH_TYPE_NONE)
+            if (Pool.Connection.Session.IsLocalSuperuser || Helpers.GetCoordinator(Pool.Connection).external_auth_type == Auth.AUTH_TYPE_NONE)

```

Comment:
This if is a bit stupid if there is only an else, should be a false test

Reply:
Will be adding a commit. Bothers me as well now that you've pointed it out